pkgs/applications/networking/remote/vmware-horizon-client/default.nix

```
@@ -32,10 +63,14 @@ let
       # Deleting the bundled library is the simplest way to force it to use our version.
       rm -f "$out/lib/vmware/gcc/libstdc++.so.6"
 
+      # This libjpeg library interferes with Chromium, so we will be using ours instead.
+      rm -f $out/lib/vmware/libjpeg.*
```

```suggestion
      rm $out/lib/vmware/libjpeg.*
```

Otherwise removal of those files does not error.

---

I removed the `-f` from both that, and the above `rm` commands. It seems to work, so seems fine by me.